docs/sources/docker-hub-enterprise/quick-start.md

```diff
@@ -45,7 +45,7 @@ You should be able to complete this guide in about thirty minutes.
 > **Note:** This guide assumes you are familiar with basic Docker concepts such
 > as images, containers, and registries. If you need to learn more about Docker
 > fundamentals, please consult the
-> [Docker user guide](http://docs.docker.com/userguide/).
+> [Docker user guide](https://docs.docker.com/userguide/).
```

Comment:
This file is copied from one of our private repos which you don't have access to.  The change actually has to be made there or else the next copy over will overwrite your change. ping @SvenDowideit @fredlf for the fix on dhe repo.